src/js/actions/layers.js

```diff
@@ -2413,49 +2414,6 @@ define(function (require, exports) {
     };
 
     /**
-     * create a vector mask on the selected layer
-     *
-     * @return {Promise}
-     */
-    var addVectorMask = function () {
```

Comment:
@pineapplespatula: is it OK to remove this? Or are you planning to use it sometime soon?